docker/Dockerfile

```diff
@@ -21,7 +21,7 @@
 #
 #############################################################################
 
-FROM debian:testing
+FROM debian:bookworm
 LABEL maintainer="Andras Mitzki <andras.mitzki@balabit.com>, László Várady <laszlo.varady@balabit.com>"
```

Comment:
Please also update the maintainer label

Reply:
I also added the copyright update to the file